ft_defaults.m

```diff
@@ -147,6 +147,20 @@
   prevcleanup = {};
 end
 
+% show a welcome message
+fprintf([
+  '-------------------------------------------------------------------------------------------' ...
+  '\nFieldTrip is developed by members and collaborators of the Donders Institute for Brain,' ...
+  '\nCognition and Behaviour at the Radboud University, Nijmegen, the Netherlands.' ...
```

Comment:
I think that 'the' can be omitted

Reply:
you mean the 2nd 'the' for Radboud?

Reply:
yes, I think Radboud University is without 'the'